docs/tasks.md

```diff
@@ -387,9 +387,22 @@ and the [`Workspaces` in a `TaskRun`](../examples/v1beta1/taskruns/workspace.yam
 
 ### Emitting results
```

Comment:
/lgtm

one more section on `Consuming Results` might be helpful as well, emphasizing that the consumer task ends up having dependency on the emitting results task and executed after the dependencies are executed successfully.

(if not documented anywhere 🙏 )

Reply:
There are a couple of places where it looks like we describe how results appear:

- The [TaskRun](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md#monitoring-results) doc describes a bit about their appearance in the Status section of the resource.
- The [Pipeline](https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md#configuring-execution-results-at-the-task-level) doc describes a bit about referencing results between tasks and mentions the task ordering.

I think that Pipeline doc could use a bit of work to make it a tad clearer. I also notice that the link from there to this doc's results section is broken.  I'll make a PR for that.  Good call, thanks @pritidesai !